nucypher/blockchain/eth/sol/source/contracts/MultiSig.sol

```diff
@@ -0,0 +1,163 @@
+pragma solidity ^0.5.3;
+
+
+import "zeppelin/math/SafeMath.sol";
+
+
+/**
+* @notice Multi-signature contract with off-chain signing
+**/
+contract MultiSig {
+    using SafeMath for uint256;
+
+    event Executed(address indexed sender, uint256 indexed nonce, address indexed destination, uint256 value);
+    event OwnerAdded(address indexed owner);
+    event OwnerRemoved(address indexed owner);
+    event RequirementChanged(uint16 required);
+
+    uint256 constant public MAX_OWNER_COUNT = 50;
+
+    uint256 public nonce;
+    uint8 public required;
+    mapping (address => bool) public isOwner;
+    address[] public owners;
+
+    // @notice Only this contract can call method
+    modifier onlyThisContract() {
+        require(msg.sender == address(this));
+        _;
+    }
+
+    function () external payable {}
+
+    /**
+    * @param _required Number of required signings
+    * @param _owners List of initial owners.
+    **/
+    constructor (uint8 _required, address[] memory _owners) public {
+        require(_owners.length > 0 &&
+            _owners.length <= MAX_OWNER_COUNT &&
+            _required <= _owners.length &&
+            _required > 0);
```

Comment:
minor detail, but there is `_owners.length > 0` when it already requires `0 < _required <= _owners.length <= MAX_OWNER_COUNT`

Reply:
👍